druid/src/widget/split.rs

```diff
@@ -65,6 +67,14 @@ impl<T> Split<T> {
         self.split_point = split_point;
         self
     }
+    /// Set the minimum size for both sides of the split
+    /// The value must be atleast 0.0
```

Comment:
```suggestion
    /// The value must be greater than or equal to `0.0`.
```

Reply:
Seems a bit clearer what happens in case of 0.0 indeed :)